code/modules/clothing/glasses/thermals.dm

```diff
@@ -38,10 +39,11 @@
 	icon_state = "eyepatch"
 	item_state = "eyepatch"
 	body_parts_covered = 0
+	species_restricted = null
```

Comment:
Any reason you have this?

Reply:
its to allow diona to wear them and bypass the restriction of all the thermal subtypes

Reply:
Ah gotcha. The collapsed rest of the file made it difficult to see in context. Seems good to me.